autoload/ale/linter.vim

```diff
@@ -13,6 +13,7 @@ let s:default_ale_linter_aliases = {
 \   'Dockerfile': 'dockerfile',
 \   'csh': 'sh',
 \   'plaintex': 'tex',
+\   'rmarkdown': 'r',
```

Comment:
Update the documentation as mentioned above, and I will merge this.

Reply:
Added as you suggested, I think that the travis failure is outside of my control:

https://travis-ci.com/w0rp/ale/jobs/206776131#L450